server/services/messaging/message_service.py

```diff
@@ -181,9 +194,46 @@ def has_user_new_messages(user_id: int) -> dict:
         return dict(newMessages=new_messages, unread=count)
 
     @staticmethod
-    def get_all_messages(user_id: int):
+    def get_all_messages(user_id: int, locale: str, page: int, page_size=10, sort_by=None, sort_direction=None, message_type=None, from_username=None, project=None, task_id=None):
         """ Get all messages for user """
-        return Message.get_all_messages(user_id)
+        sort_by = sort_by or "date"
+        sort_direction = sort_direction or "desc"
+        query = Message.query
+
+        if project is not None:
+            query = db.session.query(Message, ProjectInfo) \
+                              .filter(Message.project_id == ProjectInfo.project_id) \
+                              .filter(ProjectInfo.locale == locale) \
```

Comment:
Quick fix: could we check for both the user-supplied locale as well as `en`? (i.e. `ProjectInfo.locale.in_([locale, 'en'])`) Whenever I ran a query on the API page that was the same as in my inbox, I got no results because my browser sends `en-US`.